packages/react-components/src/components/Picker/PickerListItem.tsx

```diff
@@ -33,7 +33,44 @@ export const PickerListItem: React.FC<IProps> = ({
       );
     }
 
-    return item.name;
+    return (
+      <>
+        {item.showCheckbox && (
```

Comment:
How this `showCheckbox` option is working? From what I can see, it will always be unchecked. And how it is related to our already present check icon? 
![image](https://github.com/livechat/design-system/assets/12149393/7326af28-6809-4613-b700-3201d223f4e3)

Reply:
Checkbox should be in "checked" state if this option is selected

Reply:
thanks, good points! I changed the logic so that the check icon is not present together with checkbox and it is checked when option is selected - https://github.com/livechat/design-system/pull/741/commits/635dfcd0246ada4d178c3f055fad2d534273f666